src/util/saveArticles.ts

```diff
@@ -118,26 +118,62 @@ async function downloadBulk(listOfArguments: any[], downloader: Downloader): Pro
                     return resp;
                 });
             },
-            {concurrency: CONCURRENCY_LIMIT}
+            { concurrency: CONCURRENCY_LIMIT }
         );
     } catch (err) {
         logger.log(`Not able download in bulk due to ${err}`);
     }
 }
 
+async function getAllArticlesToKeep(downloader: Downloader, mw: MediaWiki, dump: Dump) {
+    if (!dump.customProcessor) {
```

Comment:
I would like to see this specific test outside the `getAllArticlesToKeep()` call.
Here, in place, I would like to check if `dump.customProcessor.shouldKeepArticle` function exists at all. We can have a customflavour which does not implement that function.